x/msg_authorization/alias.go

```diff
@@ -0,0 +1,25 @@
+package msg_authorization
+
+import (
```

Comment:
Note you can auto-generate this, like they did in most of the examples in the sdk. I had to reverse engineer how and then documented it here:

https://github.com/regen-network/onboarding/blob/master/module_layout.md#aliases

(Sorry, I should have shared this doc a while ago, it captured my learnings on the sdk. Please add prs if you want to add more info)

Reply:
Will check this out, will make sure my code is in accordance with all the suggestions

Reply:
Looks like you resolved this. Nice.